editors/code/package.json

```diff
@@ -433,6 +443,11 @@
                     "type": "object",
                     "default": {},
                     "description": "Optional settings passed to the debug engine. Example:\n{ \"lldb\": { \"terminal\":\"external\"} }"
+                },
+                "rust-analyzer.debug.useLaunchJson": {
```

Comment:
I'd say this should just always be `true`

Reply:
Do you mean that we do not need this option at all?

Reply:
Yes, I don’t think it is needed